spec/models/item_spec.rb

```diff
@@ -5,4 +5,14 @@
     expect(create(:item)).to be_valid
   end
 
+  it 'has a status of OOS if condition is OOS' do
+    item = build :item, condition: 'Out of Service', status: 'Unassigned'
+    expect(item.status).to eq('Out of Service')
+  end
+
+  it 'has a status of Unassigned if condition is Available and status is Unassigned' do
+    item = build :item, condition: 'Available', status: 'Unassigned'
+    expect(item.status).to eq('Unassigned')
```

Comment:
this doesn't look wacky, but it does look like you're just testing that FactoryGirl's `build` method did what you asked it too.

the fact that it could be something OTHER than what you told factory girl to build is again VERY surprising (in a bad way) I expect that if i set the status to "foo" the status should stay foo when i query it. Maybe a `before_save` type hook might change it, but there hasn't been anything to trigger something like that. 

I really don't like this behavior.